chapters/planning-data-work.tex

```diff
@@ -388,11 +388,9 @@ \subsection{Organizing files and folder structures}
 
 The \texttt{DataWork} folder may be created either inside
 an existing project-based folder structure, or it may be created separately.
-It's usually created by the leading RA in agreement with the PI.
-Increasingly, our recommendation is to create the \texttt{DataWork} folder
-separately from the project management materials,
-reserving the ``project folder'' for contracts, Terms of Reference, briefs and other administrative or management work.
-  \index{project folder}
+It is preferable to create the \texttt{DataWork} folder
+in a code folder maintained separately from the project management materials 
```

Comment:
```suggestion
separately from the project management materials 
```
I am not sure what "in a code folder" means. It should just be separate, right? And in the lines below we explain what we mean by this.